docs/reference/aggregations/bucket/datehistogram-aggregation.asciidoc

```diff
@@ -52,15 +153,16 @@ POST /sales/_search?size=0
 // CONSOLE
 // TEST[setup:sales]
 
-==== Keys
+===== Keys
 
 Internally, a date is represented as a 64 bit number representing a timestamp
-in milliseconds-since-the-epoch. These timestamps are returned as the bucket
+in milliseconds-since-the-epoch (01/01/1970). These timestamps are returned as
```

Comment:
I think we should clarify that the epoch is **UTC midnight** on January 1 1970.

Reply:
done